### ISSUE TITLE
Multi Validation pipeline

### ISSUE THREAD

**Issue by wematan** [Deci-AI/super-gradients#1014]
### Is your feature request related to a problem? Please describe.
Sometimes it makes sense to use multiple validation sets in order to test your model, 
for example a validation set for each use-case (night, day, small objects, large objects etc...) 

### Describe the solution you'd like
The trainer should be able to accept multiple validation dataloader and run the validation loop on each and report the metrics connfigured in the config file.

**Comment by shaydeci** [comment]
Hello @wematan, assuming your datasets output the same format- isn't [ConcatDataset](https://pytorch.org/docs/stable/data.html#torch.utils.data.ConcatDataset) a solution?
You can just concatenate the different datasets, and instantiate a data loader for the concatenated dataset.

**Comment by BloodAxe** [comment]
I think the ideas was to obtain validation metrics per dataset:
{ 
   "night": { "mAP": 0.123 },

**Comment by wematan** [comment]
Hi @shaydeci,

@BloodAxe is correct...
the idea is to get the metrics for each dataset segment.

BR,
Matan.

**Comment by wematan** [comment]
Thanks,
I'll try that solution then.
